src/promptflow-core/promptflow/core/_errors.py

```diff
@@ -170,9 +176,23 @@ class InvalidSampleError(CoreError):
     pass
 
 
+class ChatAPIFunctionRoleInvalidFormat(CoreError):
```

Comment:
Does prompty support functions besides tools?

Reply:
Yes.

Reply:
yes, prompty support functions and tools